### ISSUE TITLE
[3.x] Use /twill as default in Twill 3

### ISSUE THREAD

**Issue by haringsrob** [area17/twill#1867]
In Twill 3 we want to use /twill as the default cms path.

This will speedup the initial setup and will require less setup for most use-cases.

**Comment by haringsrob** [comment]
kept /admin

**Comment by antonioribeiro** [comment]
Why, @haringsrob ?

**Comment by haringsrob** [comment]
Hey @antonioribeiro what is the question exactly?

**Comment by antonioribeiro** [comment]
I'm asking about your last message "kept /admin", which I believe you didn't change it to `/twill`, why?

**Comment by ifox** [comment]
The changes currently in 3.x might create some confusion, so let's discuss.

At the moment, migrating from 2 to 3, the admin ends up being served at `/admin`, even when using a subdomain. I don't think the path should default to `/admin`, or `/twill` for that matter, when using the subdomain approach.

**Comment by ifox** [comment]
Discussed with @haringsrob on Discord, what he is describing above is the expectation so he will check and add tests for it.

@antonioribeiro I'm curious to hear what was your take on this?

**Comment by antonioribeiro** [comment]
I was really asking about not using `/twill`. If we want the brand to be known and remembered, it's better to use `twill` everywhere, and the URL is something people use a lot. After we go into the dashboard the only place where we see the Twill name is the `Made with Twill` at the bottom of the page where probably no one read.

On the config side, I was a bit confused recently when I configured 

``` dotenv
ADMIN_APP_URL=https://twill.domain.com
```

And I was forced to go to `https://twill.domain.com/admin`.
 
In order for it to drop the `/admin` I had to add this to my `.env` file:

``` dotenv
ADMIN_APP_PATH=
```
